content/reference/v1beta1/application.md

```diff
@@ -164,3 +165,12 @@ The format of the Y axis labels with support for all Grafana [units](https://gra
 
 ### yAxisTemplate
 Y axis labels template. Use `{{ value }}`.
+
+## targetKotsVersion
+The KOTS version that is targeted by the release.
+
+Specifying this in the application spec of the release will enforce compatibility checks for new installations and block the installation if the KOTS version being used is greater than the targeted KOTS version.
```

Comment:
On the initial install, does it not check for an exact version match?

Reply:
no, it will check if it's less than or equal to the target version.

Reply:
target kots version was originally "max kots version" but we renamed it because it doesn't really block app updates (and for other reasons too), so it makes sense this way.